### PR TITLE
NP-4190 Reduce CPU Usage

### PR DESCRIPTION
**Pull request by Ramjii**

### CODE REVIEW

File: nanome/_internal/_plugin_instance.py [hunk 1]
```diff
@@ -17,8 +21,16 @@ def _call(self, id, *args):
     def _run(self):
         try:
             self.start()
+            last_update = timer()
             while self._network._receive():
                 self.update()
+
+                current_time = timer()
+                dt = last_update - current_time
+                sleep_time = min(UPDATE_RATE - dt, MINIMUM_SLEEP)
```
Comment (EthanVanderHorn) [inline nanome/_internal/_plugin_instance.py]: use max

File: nanome/_internal/_plugin_instance.py [hunk 2]
```diff
@@ -17,8 +21,16 @@ def _call(self, id, *args):
...
+                last_update = current_time
```
Comment (EthanVanderHorn) [inline nanome/_internal/_plugin_instance.py]: put last_update = timer() after sleep

File: nanome/_internal/_plugin_instance.py [hunk 3]
```diff
@@ -3,7 +3,11 @@
 from nanome._internal._network._commands._callbacks import _Messages
```
Comment (EthanVanderHorn) [inline nanome/_internal/_plugin_instance.py]: Test with MD

Comment (Ramjii) [reply nanome/_internal/_plugin_instance.py]: MD became super slow, but this problem is already on master or 0.0.3

**Pull request merged**